qf_lib/plotting/charts/pie_chart.py

```diff
@@ -0,0 +1,68 @@
+#     Copyright 2016-present CERN – European Organization for Nuclear Research
+#
+#     Licensed under the Apache License, Version 2.0 (the "License");
+#     you may not use this file except in compliance with the License.
+#     You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+from typing import Tuple
+
+import numpy as np
+
+from qf_lib.containers.series.qf_series import QFSeries
+from qf_lib.plotting.charts.chart import Chart
+
+
+class PieChart(Chart):
```

Comment:
Let's add this to our API docs. For that you need to include this into the:
qf-lib > docs > source > plotting.rst into the charts part